Automattic/wp-calypso
# Add affiliates cards to the My Home screen

Related to #

## Proposed Changes

This adds task and educational cards for the Affiliates Marketing Program.
* the card (with a graph illustration) will be visible in the primary slider for the Post Launch view
* the card (with an icons illustration) will be visible in the secondary slider for other views

More info: peRNfl-IS-p2
Copy and designs: Automattic/Dotcom-Creative-Requests#172


## Testing Instructions



* Sandbox `public-api.wordpress.com`
* Apply D151274-code
* Go to `/home/{SITE_ID}?view=VIEW_SITE_SETUP`
* You should see the Affiliates slide in the secondary slider
<img width="460" alt="Screenshot 2024-06-06 at 10 58 18" src="https://github.com/Automattic/wp-calypso/assets/2749938/69b8a019-d869-4a70-b3ab-1d5340685846">

* Go to `/home/{SITE_ID}?view=VIEW_POST_LAUNCH`
* You should see the Affiliates slide in the primary slider 
<img width="460" alt="Screenshot 2024-06-06 at 10 59 54" src="https://github.com/Automattic/wp-calypso/assets/2749938/d55d46b8-a37e-40e4-88fe-a2023f2164a3">

* Switch to a non-English locale
* The Affiliates slide should not be visible for non-translated locales


## Pre-merge Checklist
